contrib/netperf/collectd-td.conf

```diff
@@ -27,9 +27,9 @@ WriteQueueLimitLow 5000
 	ReportByConnections true
 	ReportByPorts false
 
-	TcpInfoField "rtt  GAUGE  uint32:68  rcvd_ack"
-	TcpInfoField "total_retrans  GAUGE uint32:100"
-	TcpInfoField "rttvar  GAUGE uint32:72   rcvd_ack"
+	TcpInfoField "rtt		GAUGE   uint32:68   rcvd_ack"
+	TcpInfoField "delta_retrans	COUNTER uint32:100"
```

Comment:
That's not the type I think appropriate (though we analyze it on our own etc.) - I think it should be GAUGE. In that case you'll need to enter some "delta" functionality to the field there instead of assuming all COUNTERs will behave this way. 
https://collectd.org/wiki/index.php/Data_source specifies that "...four data source types which are basically identical to the data source types of the same name in RRDtool... For a description of data source types in RRDtool please refer to the rrdcreate(1) manual page".
From http://linux.die.net/man/1/rrdcreate it says "COUNTER is for continuous incrementing counters like the ifInOctets counter in a router. The COUNTER data source assumes that the counter never decreases, except when a counter overflows..."